llama_stack/providers/tests/vector_io/conftest.py

```diff
@@ -51,7 +51,7 @@
     ),
     pytest.param(
         {
-            "inference": "bedrock",
+            "inference": "ollama",
```

Comment:
I am not sure about this change. Let's wait for others to confirm here

Reply:
That makes sense to me.  If someone understands why this was the way it was and why it wasn't working for me, that would be very helpful.